p5/sketch/util.py

```diff
@@ -9,3 +9,8 @@ def read_shader(filename):
 
 def ensure_p3d(name):
     assert p5.mode == 'P3D', name + "is only available in P3D renderer"
+
+def scale_colors(r, g, b):
```

Comment:
Can we use another name for this? Its not clear what scale is going to be applied. And it ignores the alpha channel.

Reply:
would `div_by_255` work? I guess we can also call it `unbounded_normalize`.

Reply:
We can use `div_by_255`

Reply:
Or add a default attribute `scale_tuple(tuple, scale=255):`